Packs/CommonScripts/ReleaseNotes/1_3_17.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### RunPollingCommand
+- Fixed script inputs parsing. 
```

Comment:
```suggestion
- Fixed an issue where the script was not accepting integer inputs.
```

Reply:
It is not only this actually, also added encoding for the additional args inputs such that non-English chars could be accepted in the additional args.